### DISCUSSION TITLE
New plugin pods-admin-columns

### DISCUSSION THREAD

**Discussion by joker-x** [pods-framework/pods#6714] [Ideas]
I have written a plugin that add this Pods Admin UI configuration option to any custom field to allow show it as a column in Wordpress dashboard: 

![image](https://user-images.githubusercontent.com/1895563/140832251-3639e488-8c14-40b9-9d4b-97569fab09bd.png)

It is my first plugin. Any help and advices are wellcome.

Source code: https://github.com/joker-x/pods-admin-columns

**Comment by sc0ttkclark** [comment]
Would you be interested in contributing this as an enhancement to Pods itself or would you prefer to keep it in it's own plugin that you plan to expand with further functionality?

If you would like to contribute, maybe we can find some time to hang out for about an hour this month and we can pair program as we get it incorporated within Pods 2.8.5?

**Comment by joker-x** [comment]
I would like to integrate my little plugin in pods core. As independent plugin, it could be used by hundreds of wordpress instances. As an enhancement, by hundreds of thounsand.

I am very excited to do a pair program session with you. Could I invited some friends to it?

Thanks a lot!

**Comment by sc0ttkclark** [comment]
Sounds great to me, message me on the Pods slack and we can get this going. I have availability next week on my schedule to do a zoom dev session with you to integrate it.

https://support.pods.io/chat/ for Slack access

**Comment by sc0ttkclark** [comment]
Hit me up if you're still interested in having this go into Pods @joker-x

**Comment by xenio** [comment]
This is very helpful, still waiting every release to be included in Pods.
